### PR TITLE
Save email and timestamp to the database.

### PR DESCRIPTION
**Pull request by lawkaringithi**
Other user details are yet to be structured.

### CODE REVIEW

File: app/components/page-header.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-				feedback: this.get('feedback'),
-				createdAt: DS.attr('date', {
-					defaultValue() { return new Date(); }
-				})			
+				// shop: this.get('shop'),
+				// feedback: this.get('feedback'),
+				createdAt: new Date()
 			});
-			newSubcriber.save()
+			newSubcriber.save();
```
Comment (bomb1e) [inline app/components/page-header.js]: Does this work?

Comment (lawkaringithi) [inline app/components/page-header.js]: What exactly?

Comment (bomb1e) [inline app/components/page-header.js]: Haven't seen any records in the db

Comment (lawkaringithi) [inline app/components/page-header.js]: Oh, I just deleted them after testing

Comment (bomb1e) [inline app/components/page-header.js]: Ok cool

**Pull request merged**